source/sass/blocks/field.scss

```
@@ -1,18 +1,60 @@
+.field__label {
+  align-self: center;
+  margin-right: 12px;
+  color: $color-grey-600;
+
+  @media (min-width: $tablet-width) {
+    font-size: 20px;
+    line-height: 30px;
+  }
+}
+
 .field__input {
+  position: relative;
   text-transform: uppercase;
   font-weight: 400;
-  font-size: 20px;
-  line-height: 30px;
+  font-size: 16px;
+  line-height: 18px;
   color: $color-grey-600;
   margin: 0;
   font-family: $font-second;
+  min-height: 45px;
+  padding-left: 15px;
+  border: 1px solid $color-grey-150;
+  width: 150px;
+  flex-grow: 1;
+
+  @media (min-width: $tablet-width) {
+    font-size: 20px;
+    line-height: 30px;
+    padding-left: 20px;
+
+    .field__input::after {
```

`.field__input` вложен в `.field__input` у тебя здесь.
Чтобы сослаться на самого себя в вложенном классе пишется так:
```suggestion
    &::after {
```

---

все равно псеводоэлемента нет